### PR TITLE
Basic Text display for bone set

### LINKED ISSUE #15: Add HTMX-based Image Upload with Preview
- **Description:**
Use HTMX to add dynamic behavior to the image upload form in the Flask app. This will allow users to upload an image and immediately see a preview without needing to reload the page.

- **Tasks:**
- [ ] Add an image upload form in the existing HTML template.
- [ ] Integrate HTMX to send the image data to the backend without refreshing the page.
- [ ] Create a route in Flask that returns an image preview upon upload success.
- [ ] Display the uploaded image on the same page using HTMX response handling.

- **Why It's Useful:** 
Improves user experience by giving immediate feedback to the user, making the process smoother and more interactive.
- **Effort**: 2.5/5.
- **Tech Stack/Tool:** HTMX (for frontend interactivity), Flask (for backend image handling), HTML/CSS.
- **Newcomer Friendly?:** Yes, HTMX is relatively simple to learn and the task is clearly defined with a focus on frontend behavior.
- **Clearly Defined?:** Yes, the tasks break down exactly what needs to happen step-by-step, from form creation to image preview.

### PR DESCRIPTION
**Pull request by tluke900**
Fixes #15 

Basic HTML layout created, displaying example layout of upper-limb bone set. 

HTML for bone displays.

Links not created yet, initial commit for Proof of concept.

### CODE REVIEW

**Comment by kate-holdener** [comment]
@tluke900 That's a nice start with placeholder for text. The idea is to not hard-code any information, but pull it dynamically from the data branch of the repo, where the information is defined in the JSON files.

**Review by jacksayshi — CHANGES_REQUESTED** [review]
Its is a nice structure. Clean and simple. Easy to read and understand. Now lets build on this. We need our application to pull data from the data branch (where our data exists). They will be stored as json files. 
They should be fetched and displayed in the format you created.

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                <li>Slightly longer than the radius, always ends on the side of the pinky.</li> -->
+
+
+            <!-- </div> --->
+         </div>
+    </div>
+    <script>
+
+        document.addEventListener("DOMContentLoaded", function () {
+            fetch('/Data_Directory/boneSet.json')
```
Comment (kate-holdener) [inline templates/index.html:65]: You should be fetching from here: https://github.com/oss-slu/DigitalBonesBox/blob/data/databones/json/boneset.json

**Comment by jacksayshi** [comment]
I tried to run your code. But the HTML page was displaying as a blank page. Please try running it again on your side.

**Review by tluke900 — COMMENTED** [review]
Should be fixed to address issues.

**Review by jacksayshi — APPROVED** [review]

**Pull request merged**